gitlab-runner/buildscripts/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+CHANGED_FILES=`git diff --name-only master...${TRAVIS_COMMIT}`
+INFRA=True
+DIR="gitlab-runner"
+
+for CHANGED_FILE in $CHANGED_FILES; do
+  if [[ "$CHANGED_FILE" =~ $DIR ]]; then
+    INFRA=False
+    break
+  fi
+done
+
+if [[ $INFRA == True ]]; then
+  echo "NOT building Gitlab-runner infra image."
+  
+else
+  echo "building Gitlab-runner infra image."
+  cd ..
+  docker build -t atulabhi/kops:v22 .
```

Comment:
Should this be?
```suggestion
  docker build -t openebs/infra:v22 .
```

Reply:
maybe we can use latest/ as other images in test-tools use this. Also, we will not have to change in the gitlab ci yamls